feature-libs/customer-ticketing/assets/translations/en/customer-ticketing.ts

```diff
@@ -52,4 +52,10 @@ export const customerTicketing = {
     closeRequest: 'Close Request',
     ticketNotFound: 'Ticket not found.',
   },
+  myAccountV2CustomerTicketing: {
+    heading: 'Customer Service',
+    showMore: 'Show More',
+    changedOn: 'Changed On: {{value}}',
+    ticketId: 'ID: {{value}}',
+  },
```

Comment:
[QUESTION]
By placing `myAccountV2CustomerTicketing` inside the old `customerTicketing` namespace, the new labels for the `myAccountV2CustomerTicketing` will be (lazy-)loaded together with all the other customer-ticketing (non-v2!) labels. My question is: was it a deliberate choice or just accidental? 

Why I'm asking: I can see that for the _order_ feature we're loading `v2` translations separaterly from `non-v2`.

Reply:
done